arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3820,19 +3863,17 @@ arangodb::Result listHotBakupsOnCoordinator(
 arangodb::Result deleteHotBakupsOnCoordinator(
   VPackSlice const payload, VPackBuilder& report) {
 
-  std::vector<std::string> listIds, deleted;
+  std::unordered_map<std::string, BackupMeta> listIds;
+  std::vector<std::string> deleted;
   VPackBuilder dummy;
   arangodb::Result result;
 
   ClusterInfo* ci = ClusterInfo::instance();
   std::vector<ServerID> dbServers = ci->getCurrentDBServers();
 
-  result = hotBackupList(dbServers, payload, listIds, dummy);
-  if (!result.ok()) {
-    return result;
-  }
+  std::string id = payload.get("id").copyString();
 
-  result = removeLocalBackups(listIds.front(), dbServers, deleted);
+  result = removeLocalBackups(id, dbServers, deleted);
   if (result.ok()) {
```

Comment:
Is this supposed to be "!result.ok()"?  As written, an ok() means no report.

Reply:
Fixed.